readme.md

```diff
@@ -263,6 +264,51 @@ namespace Azure.Service.Models
 
 </details>
 
+### Preserve raw Json value of a property
+
+Use the [Change a model property type](#Change-a-model-property-type) approach to change property type to `JsonElement`.
+
+<details>
+
+**Generated code before (Generated/Models/Model.cs):**
+
+``` C#
+namespace Azure.Service.Models
+{
+    public partial class Model
+    {
+        public string Property { get; }
```

Comment:
If the property, parameter, or return value is typed as an object instead of a string, will this work as well?

Reply:
This is only for properties for now but yes, it would work with any type of original property.